src/Features/CSharp/Portable/ConvertToRawString/ConvertInterpolatedStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,685 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this is the 'meat' of the change.  it's all the new code to deal with interpolated string conversion.